tests/JSValueTests.Usage.swift

```diff
@@ -135,11 +135,11 @@ class JSValueUsageTests : XCTestCase {
                 "url" : "http://remote.bloxus.com/"
             ]
         
-        let blog = make ⇒
-            (json["id"].number ⇒ toInt) ⇒
-            json["name"].string ⇒
-            json["needspassword"].bool ⇒
-            (json["url"].string ⇒ toURL)
+        let blog = make(
```

Comment:
Apparently, the curried format is going to be [removed in the next version of Swift](https://github.com/apple/swift-evolution/blob/master/proposals/0002-remove-currying.md). Thoughts?